content/blog/2024/sass-compiler.md

```diff
@@ -52,29 +63,30 @@ and `sass-embedded` exposes the same API, but around a native Dart executable.
 In many situations, `sass-embedded` is faster.
 {% endcallout %}
 
-3. In your `vite.config.js` file, set `css.preprocessorOptions.scss.api` to `modern-compiler`.
+3. Vite 7.0 made the Sass Compiler API the default (and removed support for the
+   non-compiler API), so you don't need to opt in to the API. For Vite 5.4.0
+   through < 7.0, you will need to opt in. In your `vite.config.js` file, set
+   `css.preprocessorOptions.scss.api` to `modern-compiler`.
 
 ```js
 ...
 css: {
   preprocessorOptions: {
+    // Set `sass` if using the indented syntax
     scss: {
+      // Only needed on versions < 7.0.0
       api: 'modern-compiler',
     }
   }
 }
 ...
 ```
 
-{% callout %}
-If you're using the indented syntax, you'll need to use the `sass` key
-instead of `scss`.
-{% endcallout %}
-
-4. Adjust any options from the [`legacy`] API options to the [`modern`] API
-   options. In my case, I needed to update `pkgImporter` to `importers: [new
-   NodePackageImporter()]` and change the import of `NodePackageImporter` from
-  `sass` to `sass-embedded`.
+4. Vite 7.0 also removed support for the legacy API, which doesn't support the
+   Compiler API anyways. Adjust any options from the [`legacy`] API options to
```

Comment:
I find this first sentence a bit difficult to parse. Maybe the clause after the comma isn't needed?

Reply:
Good call -- updated!